src/utils/formatUtils.ts

```diff
@@ -0,0 +1,10 @@
+export const isJsonFormat = (text: string) => {
+  try {
+    const tokens = JSON.parse(text);
+
+    return tokens.hasOwnProperty("access") && tokens.hasOwnProperty("refresh");
+  } catch (error) {
+    localStorage.removeItem("jwt");
```

Comment:
i think better we have JwtUtils and have some function like saveToken, getToken, removeToken, etc